azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -123,6 +123,15 @@ private void sendSlaAlertEmail(final SlaOption slaOption, final String slaMessag
     }
   }
 
+  private String getJobOrFlowName(SlaOption slaOption) {
+    String jobName = (String) slaOption.getInfo().get(SlaOption.INFO_JOB_NAME);
+    if (org.apache.commons.lang.StringUtils.isNotBlank(jobName)) {
```

Comment:
isNotEmpty to be safer?

Reply:
Why would we want to include a "job name" that is whitespace only? I don't think that should be even possible.